dockers /Dockerfile

```
@@ -0,0 +1,25 @@
+FROM amazonlinux:2023
+RUN yum update 
+WORKDIR APP
+RUN yum install wget -y
+RUN mkdir -p /usr/share/man/man1/
+RUN wget https://download.java.net/java/GA/jdk17/0d483333a00540d886896bac774ff48b/35/GPL/openjdk-17_linux-x64_bin.tar.gz
```

add `&&` here to stop the build from progressing should `wget` fail for some reason

```suggestion
RUN wget https://download.java.net/java/GA/jdk17/0d483333a00540d886896bac774ff48b/35/GPL/openjdk-17_linux-x64_bin.tar.gz && \
tar xvf openjdk-17_linux-x64_bin.tar.gz
```